src/docfx/lib/learn/Achievement.cs

```diff
@@ -0,0 +1,16 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace Microsoft.Docs.Build
+{
+    internal class Achievement
+    {
+        public SourceInfo<string> Uid { get; set; }
+
+        public string Type { get; set; } = "";
```

Comment:
Use enum over strings.
```suggestion
        public AchievementType Type { get; set; };
```

Reply:
OK